src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -2941,21 +2940,20 @@ void RadioSpinel<InterfaceType, ProcessContextType>::LogSpinelFrame(const uint8_
             bool isARetx;
             bool skipAes;
 
-            unpacked = spinel_datatype_unpack(
-                data, len,
-                SPINEL_DATATYPE_DATA_WLEN_S                                   // Frame data
-                    SPINEL_DATATYPE_UINT8_S                                   // Channel
-                        SPINEL_DATATYPE_UINT8_S                               // MaxCsmaBackoffs
-                            SPINEL_DATATYPE_UINT8_S                           // MaxFrameRetries
-                                SPINEL_DATATYPE_BOOL_S                        // CsmaCaEnabled
-                                    SPINEL_DATATYPE_BOOL_S                    // IsHeaderUpdated
-                                        SPINEL_DATATYPE_BOOL_S                // IsARetx
-                                            SPINEL_DATATYPE_BOOL_S            // SkipAes
-                                                SPINEL_DATATYPE_UINT32_S      // TxDelay
-                                                    SPINEL_DATATYPE_UINT32_S, // TxDelayBaseTime
-                &frame.mPsdu, &frame.mLength, &frame.mChannel, &frame.mInfo.mTxInfo.mMaxCsmaBackoffs,
-                &frame.mInfo.mTxInfo.mMaxFrameRetries, &csmaCaEnabled, &isHeaderUpdated, &isARetx, &skipAes,
-                &frame.mInfo.mTxInfo.mTxDelay, &frame.mInfo.mTxInfo.mTxDelayBaseTime);
+            unpacked = spinel_datatype_unpack(data, len,
+                                              SPINEL_DATATYPE_DATA_WLEN_S                        // Frame data
+                                                  SPINEL_DATATYPE_UINT8_S                        // Channel
+                                                      SPINEL_DATATYPE_UINT8_S                    // MaxCsmaBackoffs
+                                                          SPINEL_DATATYPE_UINT8_S                // MaxFrameRetries
+                                                              SPINEL_DATATYPE_BOOL_S             // CsmaCaEnabled
+                                                                  SPINEL_DATATYPE_BOOL_S         // IsHeaderUpdated
+                                                                      SPINEL_DATATYPE_BOOL_S     // IsARetx
+                                                                          SPINEL_DATATYPE_BOOL_S // SkipAes
+                                                                              SPINEL_DATATYPE_UINT32_S, // TxTime
+                                              &frame.mPsdu, &frame.mLength, &frame.mChannel,
+                                              &frame.mInfo.mTxInfo.mMaxCsmaBackoffs,
+                                              &frame.mInfo.mTxInfo.mMaxFrameRetries, &csmaCaEnabled, &isHeaderUpdated,
+                                              &isARetx, &skipAes, &frame.mInfo.mTxInfo.mTxTime);
```

Comment:
I think this will break forward/backward compatibility between RCP and host.
- Host being older with newer RCP or other way round.

We want to keep it compatible and avoid requiring both side to be updated (unless we have a very strong reason/justification for this). 

cc: @zhanglongxia  (also see #7937).